Gruntfile.js

```diff
@@ -19,11 +19,14 @@ function process( code, filepath ) {
 		.replace( /@DATE/g, ( new Date() ).toISOString().replace( /:\d+\.\d+Z$/, "Z" ) );
 }
 
+// Prevents CRLF EOL on Windows environments
```

Comment:
Might be worth mentioning that this is important for style checks after the concat, otherwise seems a bit arbitrary